db/seeds.rb

```diff
@@ -1,7 +1,7 @@
-Category.destroy_all
-Item.destroy_all
-User.destroy_all
-Order.destroy_all
+# Category.destroy_all
```

Comment:
@samuelssnider won't we need these calls in there? or we could end up with duplicate records in a table. I think that if we have these in the seed file we won't have to drop our table anytime that we want to reseed.